doltpy_integrations/metaflow/examples/hospital_bounty/blog_post.md

```diff
@@ -0,0 +1,164 @@
+
+At the level of the Flow, the integration looks like this:
+
+![Flow level view](dolt-metaflow-integration-flow-level.png)
+
+Drilling into one of the runs:
+
+![Run level view](dolt-metaflow-integration-run-level.png)
+
+This is all abstract, so let's install a few dependencies, grab a dataset, and get stuck some armchair data science.
+
+## Setup
+Let's get the boring stuff out of the way. We need the following:
+- Dolt and `doltpy-integrations` installed
+- Metaflow installed
+- the sample dataset we will use, which can easily clone from DoltHub
+
+### Install Dolt
+The first step is to install Dolt on a `*nix` system:
+```
+sudo bash -c 'curl -L https://github.com/dolthub/dolt/releases/latest/download/install.sh | bash'
+```
+
+There are Windows distributions and a Homebrew cask. Find more details about installation [here](https://docs.dolthub.com/getting-started/installation).
+
+### install doltpy-integrations[metaflow]
+Next let's install the Metaflow + Dolt integration. It comes packaged with both Metaflow, and Dolt's Python API, Doltpy. It's easy enough to install via `pip`:
+```
+pip install doltpy-integraions[metaflow]
+```
+
+### Get The Data
+The final step is to acquire the dataset. Dolt is a SQL database with Git-like version control features. That includes the ability to clone a remote to your local machine. We do that here:
+```
+$ dolt clone dolthub/hospital-price-transparency && cd hospital-price-transparency
+
+```
+
+Note this dataset is nearly 20 gigabytes, and could take a few minutes to clone. Once it's landed it's straightforward to jump write into SQL:
+```
+$ dolt sql
+# Welcome to the DoltSQL shell.
+# Statements must be terminated with ';'.
+# "exit" or "quit" (or Ctrl-D) to exit.
+hospital_price_transparency> show tables;
++-----------+
+| Table     |
++-----------+
+| cpt_hcpcs |
+| hospitals |
+| prices    |
++-----------+
+```
+
+Finally let's create a Dolt database for our downstream results:
+```
+$ mkdir ~/hospital-price-analysis && cd ~/hospital-price-analysis
+$ dolt init
+Successfully initialized dolt data repository.
+```
+
+Before starting to run our Metaflow flows, let's formulate a hypothesis to test.
+
+## Arm Chair Data Science
+We are going to use our example data pipeline, described at the outset, to test a hypothesis about our sample dataset. The data set we acquired is the result of a "data bounty." A data bounty is crowd sourced data gathering project sponsored by DoltHub. You can read more about this particular bounty in this blog [post](https://www.dolthub.com/blog/2021-03-03-hpt-bounty-review/).
+
+The hypothesis we wish to test is this:
+> As the observation count increases, the procedure level price variance should decrease
+
+This is a crucial question in any data collection effort. The statistical regularity of the observations, combined with the cost of each observation, very directly impacts how effective such efforts are as a means of collecting data. 
```

Comment:
i like this addition, maybe simplify this bit "...very directly impacts how effective such efforts are as a means of collecting data."